core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/ViewNames.java

```diff
@@ -111,6 +111,7 @@
     public static final String BROWN_CLUSTERS = "BROWN_CLUSTERS";
     public static final String DEPENDENCY_HEADFINDER = "DEPENDENCY_HEADFINDER";
     public static final String GAZETTEER_NE = "GAZETTEER_NE";
+    public static final String TIMEX3 = "TIMEX3";
```

Comment:
You also need to add a corresponding entry to the switch() statement for getViewType()